src/chrome/content/rules/Blogspot.com_blogs.xml

```diff
@@ -150,6 +218,79 @@
+		<test url="http://sewkindofwonderful.blogspot.com.cy/" />
+		<test url="http://sewkindofwonderful.blogspot.com.ee/" />
+		<test url="http://sewkindofwonderful.blogspot.com.eg/" />
+		<test url="http://sewkindofwonderful.blogspot.com.es/" />
+		<test url="http://sewkindofwonderful.blogspot.com.mt/" />
+		<test url="http://sewkindofwonderful.blogspot.com.ng/" />
+		<test url="http://sewkindofwonderful.blogspot.com.tr/" />
+		<test url="http://sewkindofwonderful.blogspot.com.uy/" />
+		<test url="http://sewkindofwonderful.blogspot.com/" />
+		<test url="http://sewkindofwonderful.blogspot.cz/" />
+		<test url="http://sewkindofwonderful.blogspot.de/" />
+		<test url="http://sewkindofwonderful.blogspot.dk/" />
+		<test url="http://sewkindofwonderful.blogspot.fi/" />
+		<test url="http://sewkindofwonderful.blogspot.fr/" />
+		<test url="http://sewkindofwonderful.blogspot.gr/" />
+		<test url="http://sewkindofwonderful.blogspot.hk/" />
+		<test url="http://sewkindofwonderful.blogspot.hr/" />
+		<test url="http://sewkindofwonderful.blogspot.hu/" />
+		<test url="http://sewkindofwonderful.blogspot.ie/" />
+		<test url="http://sewkindofwonderful.blogspot.in/" />
+		<test url="http://sewkindofwonderful.blogspot.is/" />
+		<test url="http://sewkindofwonderful.blogspot.it/" />
+		<test url="http://sewkindofwonderful.blogspot.jp/" />
+		<test url="http://sewkindofwonderful.blogspot.kr/" />
+		<test url="http://sewkindofwonderful.blogspot.li/" />
+		<test url="http://sewkindofwonderful.blogspot.lt/" />
+		<test url="http://sewkindofwonderful.blogspot.lu/" />
+		<test url="http://sewkindofwonderful.blogspot.md/" />
+		<test url="http://sewkindofwonderful.blogspot.mk/" />
+		<test url="http://sewkindofwonderful.blogspot.mx/" />
+		<test url="http://sewkindofwonderful.blogspot.my/" />
+		<test url="http://sewkindofwonderful.blogspot.nl/" />
+		<test url="http://sewkindofwonderful.blogspot.no/" />
+		<test url="http://sewkindofwonderful.blogspot.pe/" />
+		<test url="http://sewkindofwonderful.blogspot.pt/" />
+		<test url="http://sewkindofwonderful.blogspot.qa/" />
+		<test url="http://sewkindofwonderful.blogspot.ro/" />
+		<test url="http://sewkindofwonderful.blogspot.rs/" />
+		<test url="http://sewkindofwonderful.blogspot.ru/" />
+		<test url="http://sewkindofwonderful.blogspot.se/" />
+		<test url="http://sewkindofwonderful.blogspot.sg/" />
+		<test url="http://sewkindofwonderful.blogspot.si/" />
+		<test url="http://sewkindofwonderful.blogspot.sk/" />
+		<test url="http://sewkindofwonderful.blogspot.sn/" />
+		<test url="http://sewkindofwonderful.blogspot.tw/" />
+		<test url="http://sewkindofwonderful.blogspot.ug/" />
+
+		<!-- Causes MCB on some pages such as http://streamfare.com/telemundo.html.
+			See https://github.com/EFForg/https-everywhere/issues/8113 -->
+		<exclusion pattern="^http://2livewatch\.blogspot\.com/" />
+			<test url="http://2livewatch.blogspot.com/2016/10/telemundo-222.html" />
```

Comment:
Why not limiting the exclusion to this specific post?

Reply:
Not sure what you mean. It probably causes MCB on other paths from the same website and possibly elsewhere as well.

Reply:
I didn't found any other posts.